x-pack/test/performance/journeys/base.config.ts

```diff
@@ -23,6 +24,18 @@ export default async function ({ readConfigFile, log }: FtrConfigProviderContext
 
   log.info(` 👷‍♀️ BUILD ID ${testBuildId}\n 👷 JOB ID ${testJobId}\n 👷‍♂️ EXECUTION ID:${executionId}`);
 
+  const telemetryLabels: TelemetryConfigLabels = {
+    branch: process.env.BUILDKITE_BRANCH,
+    ciBuildId: process.env.BUILDKITE_BUILD_ID,
+    ciBuildJobId: process.env.BUILDKITE_JOB_ID,
+    ciBuildNumber: Number(process.env.BUILDKITE_BUILD_NUMBER) || 0,
+    gitRev: process.env.BUILDKITE_COMMIT,
+    isPr: !!process.env.GITHUB_PR_NUMBER,
+    prId: Number(process.env.GITHUB_PR_NUMBER) || 0,
```

Comment:
We should just not specify this label when we're not in a PR
```suggestion
    ...(process.env.GITHUB_PR_NUMBER ? { prId: process.env.GITHUB_PR_NUMBER } : {}),
```